src/Yoti.Auth/ShareUrl/Policy/DynamicPolicy.cs

```diff
@@ -17,19 +17,14 @@ public class DynamicPolicy
         [JsonProperty(PropertyName = "wanted_remember_me")]
         private readonly bool _wantedRememberMeId;
 
-        [JsonProperty(PropertyName = "wanted_remember_me_optional")]
```

Comment:
As you already mentioned to me, we may want to keep this property and set to `false`

Reply:
I've now done that, completed in 179b5be